### PR TITLE
Navbar 

### PR DESCRIPTION
**Pull request by MohamedTaha91**
- Added Test 
- Finished v1 of UI
- Added Lucide React

Navbar complete

### CODE REVIEW

File: src/components/navbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <div className="flex flex-row bg-black">
+      <Image
+        src={netflixLogo}
+        alt="Netflix Logo"
+        width={140}
+        height={35}
+        className="h-auto ml-8"
+      />
+      <menu className="flex text-small w-full gap-x-10 items-center text-white ml-5">
+        <Link href={"#"}>Inicio</Link>
```
Comment (FCartier) [inline src/components/navbar/index.tsx:17]: Why put the names in Spanish? Wouldn't it make more sense to put them in English to share your work better later on?

Comment (MohamedTaha91) [reply src/components/navbar/index.tsx:17]: we can do that no problem, it was get things moving for now, when the navbar is full functional links wise i will change the names to english
```suggestion
        <Link href={"#"}>Inicio</Link>
```

Comment (MohamedTaha91) [reply src/components/navbar/index.tsx:17]: good call! Done

Comment (FCartier) [reply src/components/navbar/index.tsx:17]: You haven't pushed your changes to this branch. Are you planning to push them with your next PR?

File: src/components/navbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        height={35}
+        className="h-auto ml-8"
+      />
+      <menu className="hidden md:flex text-small w-full h-20 gap-x-10 mr-10 items-center text-white ml-5">
+        <Link href={"#"}>Home</Link>
+        <Link href={"#"}>Series</Link>
+        <Link href={"#"}>Films</Link>
+        <Link href={"#"}>New & Popular</Link>
+        <Link href={"#"}>My List</Link>
+        <Link href={"#"}>Browse by languages</Link>
```
Comment (FCartier) [inline src/components/navbar/index.tsx:23]: Similar comment: lines 18 to 23 are almost identical. Let's apply the DRY concept ( Don't Repeat Yourself )

**Review by FCartier — CHANGES_REQUESTED** [review]
Let's apply DRY concept!

**Pull request merged**